flink-core/src/test/java/org/apache/flink/util/TimeUtilsTest.java

```diff
@@ -123,9 +154,4 @@ public void testParseDurationInvalid() {
 	public void testParseDurationNumberOverflow() {
 		TimeUtils.parseDuration("100000000000000000000000000000000 ms");
 	}
-
-	@Test(expected = IllegalArgumentException.class)
-	public void testParseDurationNumberTimeUnitOverflow() {
```

Comment:
With your patch we still possibly throw an `IllegalArgumentException` if the number overflow. Why do you remove this test?

Reply:
This test was for some removed logics in TimeUtils.

Currently this case can cause `ArithmeticException` thrown from `java.time.Duration`.
But I think we do not need it, just like we do not need to add tests for formats validation, which should be tested by `java.time.Duration` itself.